postgrid/__init__.py

```diff
@@ -470,6 +479,126 @@ def construct_event(cls, payload, secret):
             raise ValueError()
 
 
+def _av_post_request(endpoint, json=None, data=None, params=None):
+    data = _map_keys_recursive(data, _snake_to_camel)
+    params = _map_keys_recursive(params, _snake_to_camel)
+
+    headers = {'x-api-key': av_key}
+    data_to_pass = []
+    params_to_pass = []
+
+    def flatten(arr, key, value):
+        if isinstance(value, dict):
+            for inner_key, inner_value in value.items():
+                flatten(arr, f'{key}[{inner_key}]', inner_value)
+        elif isinstance(value, list):
+            for inner_value in value:
+                flatten(arr, f'{key}[]', inner_value)
+        elif isinstance(value, bool):
+            arr.append((key, 'true' if value else 'false'))
```

Comment:
Is this necessary? Wouldn't generating a JSON automatically convert these to the correct datatype? Or is this specifically for the `params`

Reply:
Yup this is for the req body and query params. Should be necessary as far as I know.